### ISSUE TITLE
Resolve identical regression tests

### ISSUE THREAD

**Issue by jdavies-st** [spacetelescope/jwst#2117]
The following two tests are identical:
```
jwst/tests_nightly/general/pipelines/test_nrs_fs_multi_spec2_3.py
jwst/tests_nightly/general/pipelines/test_nrs_fs_single_spec2.py
```
They take in the same input and produce the same output.
```
> diff test_nrs_fs_multi_spec2_3.py test_nrs_fs_single_spec2.py 
12c12
< def test_nrs_fs_multi_spec2_3(_bigdata):
---
> def test_nrs_fs_single_spec2(_bigdata):
16c16
<     using the ALLSLITS subarray and detector NRS2.
---
>     that uses a single-slit subarray (S200B1).
```

We should remove one of them or modify one to do what we think it should be doing.

**Comment by jdavies-st** [comment]
From @hbushouse, "We should modify the `test_nrs_fs_single_spec2` test to use an input exposure that uses a single-slit subarray, rather than ALLSLITS."

**Comment by jdavies-st** [comment]
Fixed in #4386
